src/components/TodoComponents/TodoForm.js

```diff
@@ -0,0 +1,39 @@
+import React from 'react';
+
+
+
+class TodoForm extends React.Component {
+    
+    constructor() {
+        super();
+        this.state = {
+            itemName: ""
+        };
+    };
+
+    handleChanges = (e) => {
+        this.setState({itemName: e.target.value});
+    };
+
+    handleSubmit = (e) => {
+        this.props.addGame(e, this.state.itemName)
+        console.log("new game added: " + this.state.itemName);
```

Comment:
Separation of concerns begs that you _handle_ ignoring the event hear in the _handler_ and let `addGame` just do what it says it does